bobcat/bb-aem-common/src/main/java/com/cognifide/qa/bb/aem/Test.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Cognifide Ltd..
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.qa.bb.aem;
+
+import com.cognifide.qa.bb.qualifier.PageObject;
+import org.openqa.selenium.WebElement;
+
+/**
+ *
+ * @author mikolaj.manski
+ */
+@PageObject
+public class Test {
+ private int x;
+ public void costam() {
```

Comment:
this is for testing purposes only it will be deleted, just wanted to check wheter it passes on travis